packages/core/bootstrap/src/lib/config/provider-limits/index.ts

```diff
@@ -103,6 +108,17 @@ const parseLimits = (limits: Limits): Limits => {
   return { http, ws }
 }
 
+export function getLastTierLimitValue(limits: Limits, rateLimit: RateLimitTimeFrame): number {
+  if (Object.keys(limits).length == 0) return 0
+  const tierList = Object.keys(limits.http)
+  if (tierList.length !== 0) {
+    const lastTier = tierList[tierList.length - 1]
+    const highestTierLimit = limits.http[lastTier][rateLimit] as number
+    return highestTierLimit
+  }
+  return 0
+}
```

Comment:
Thinking about this function some more... We don't have any guarantees that the config file is ordered, so I think we should rewrite this to iterate through all of the tiers and find the highest possible capacity.